server/views/taskList/taskList.pug

```diff
@@ -28,7 +28,15 @@ block content
         if user.role === 'CA'
             if (['UNSTARTED', 'ELIGIBILITY'].includes(licenceStatus.stage))
                 include eligibilityTask
-                include proposedAddressTask
+
+                if licenceStatus.decisions.eligible
+                    if licenceStatus.tasks.eligibility === 'DONE'
+                        include proposedAddressTask
+                        include caSubmitTask
+                else
+                    include caSubmitRefusalTask
```

Comment:
This means if the offender is ineligible for any reason it gets sent to DM. Is that correct, because it didn't use to work that way (a while since I've worked on that aspect) and it wasn't in the story.